manifests/config.pp

```diff
@@ -40,4 +40,36 @@
     owner   => 'root',
     group   => $candlepin::group,
   }
+
+  $artemis_truststore = '/etc/tomcat/artemis/truststore'
+  $artemis_truststore_password_file = 'truststore_password-file',
```

Comment:
I'd consider removing this variable altogether. In one place it's used as a filename and in the other as a cache key. If the filename changes, the cache key doesn't have to.